.gitignore

```diff
@@ -6,4 +6,5 @@ results/
 *.ipynb_checkpoints/
 dataSources/UpdateDemand/
 scripts/developer_tools/CanadaUsaTxtDiffCheck/Results.txt
-*__pycache__/
\ No newline at end of file
+*__pycache__/
+logs/

```

Comment:
Should this be src/logs/ ?

Reply:
Hmm... Im thinking just logs. I don't think we need to be uploading any log files to GitHub?

Reply:
Right, that makes sense.